### ISSUE TITLE
Help Wanted: Document how to use message attachments with incoming webhooks

### ISSUE THREAD

**Issue by mickmister** [mattermost/mattermost-developer-documentation#1115]
More context here https://github.com/mattermost/mattermost-server/issues/21036#issuecomment-1248352628

**Comment by neflyte** [comment]
Where should this information live? The linked issue says that message attachments aren't just for plugins and should be more global. We should probably have more discussion around how to handle this.

**Comment by mickmister** [comment]
@neflyte Yes it seems "Plugins" is currently being used as an umbrella section, with things that are not exclusively related to plugins, namely interactive messages in this case. https://developers.mattermost.com/integrate/getting-started. Maybe we can add a new section Interactivity for "interactive messages" and "interactive dialogs".

<img width="484" alt="image" src="https://user-images.githubusercontent.com/6913320/197287456-b3556abf-f437-4edf-a197-0987babaa167.png">

There are also message attachments in general that are currently described in https://developers.mattermost.com/integrate/reference/message-attachments. Maybe we just need to say in the webhooks section that these attachments are supported, just as they are in the posts API.

**Comment by esarafianou** [comment]
The developer docs have been migrated to [docs.mattermost.com/developers](https://docs.mattermost.com/developers). Content and code now live at [mattermost/mattermost/docs/develop](https://github.com/mattermost/mattermost/tree/master/docs/develop).  We've moved from Hugo to Docusaurus.

We're archiving this repo, so please open new issues in the [mattermost/mattermost](https://github.com/mattermost/mattermost) repo instead. There's a dedicated "Documentation" issue type there that covers both main docs and developer docs.

Closing this issue as part of the migration.
